mod_modular_vanilla/hooks/entity/world/player_party.nut

```diff
@@ -1,31 +1,30 @@
 ::ModularVanilla.MH.hook("scripts/entity/world/player_party", function (q) {
-	// MV: Modularized
+	// MV: Modularized and Changed
+	// Part of player_party.updateStrength modularization.
+	// Extracted the calculation of bro strength.
+	// Sort by strength instead of level.
+	// Added multiplier for Origin.
 	q.updateStrength = @() function()
 	{
 		this.m.Strength = 0.0;
-		local roster = ::World.getPlayerRoster().getAll();
 
-		if (roster.len() > ::World.Assets.getBrothersScaleMax())
-		{
-			roster.sort(this.onLevelCompare);
-		}
+		// Vanilla sorts by level and then checks for exceeding getBrothersScaleMax or being below getBrothersScaleMin.
+		// We sort by strength because in MV the strength may not necessarily be a linear function of Level.
+
+		local strengths = ::World.getPlayerRoster().getAll().map(@(_bro) _bro.MV_getStrength());
 
-		if (roster.len() < ::World.Assets.getBrothersScaleMin())
+		if (strengths.len() < ::World.Assets.getBrothersScaleMin())
 		{
-			// Extracted the strength of empty slot into a separate function
+			// MV: Extracted the strength of empty slot into a separate function
 			this.m.Strength += this.MV_getEmptyBroStrength() * (::World.Assets.getBrothersScaleMin() - roster.len());
 		}
 
-		foreach (i, bro in roster)
-		{
-			if (i >= ::World.Assets.getBrothersScaleMax())
-			{
-				break;
-			}
+		// sort descending
+		strengths.sort(@(_a, _b) -a <=> -_b);
```

Comment:
Can be part of an unconditional `else` to save some performance.
No need to sort anything, if we are then taking all elements from it anyways

Reply:
I've added a condition for sort and rewritten the code to use a slice and reduce intead of the for loop. Please take a look again.